components/raftstore-v2/src/operation/bucket.rs

```diff
@@ -397,9 +388,9 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
         if !ctx.coprocessor_host.cfg.enable_region_bucket() {
             return None;
         }
-        let bucket_update_diff_size_threshold = ctx.coprocessor_host.cfg.region_bucket_size.0 / 2;
+        let region_bucket_max_size = ctx.coprocessor_host.cfg.region_bucket_size.0;
```

Comment:
Seems this should be region_bucket_size * 2, so after split, the bucket size is region_bucket_size. From `region_bucket_size`'s comment, I see the expected bucket size is about region_bucket_size * 1.5.

Reply:
yes，it should be the twice than the `region_bucket_size`.